trestlebot/tasks/sync_cac_content_task.py

```diff
@@ -106,6 +402,19 @@ def _update_compdef(
                     logger.info(f"Start to update props of {component.title}")
                     compdef.components[index].props = oscal_component.props
                     updated = True
+                # The way to check control implementations needs to be updated
+                if (
```

Comment:
I might have missed it somewhere else in the code, but is there logic that prevents UUID regeneration if the control implementation or implemented requirements have not been change? If not, I think this will always be true.

Reply:
That's true, that's why I added a comment in L405 and was planning to update in a separate PR, it needs some time to investigate on how to check if there is no data changes in this nested data structure.  I see some functions in trestle ModelUtils may be helpful for this checking.

Reply:
Thanks @qduanmu. I misread the comment and didn't realize it was a `TODO` item. I understand now.